packages/core/test/financial-templates/perpetual-multiparty/PerpetualLiquidatable.js

```diff
@@ -1864,13 +1864,11 @@ contract("PerpetualLiquidatable", function(accounts) {
     const USDCScalingFactor = toBN("1000000000000"); // 1e12
 
     // By dividing the pre-defined parameters by the scaling factor 1e12 they are brought down from 1e18 to 1e6
-    const USDCPricePerToken = pricePerToken.div(USDCScalingFactor); // 1.5e6
-    const USDCDisputePrice = settlementPrice.div(USDCScalingFactor); // 1.0e6
     const USDCAmountOfCollateral = amountOfCollateral.div(USDCScalingFactor); // 150e6
-    // Note: the number of synthetics does not get scaled. This is still the 100e18 as with other tests.
+    const USDCAmountOfSynthetic = amountOfSynthetic.div(USDCScalingFactor); // 150e6
```

Comment:
This is the main change: synthetic amounts use same precision as collateral, and prices are now always expressed with 18 decimals